### DISCUSSION TITLE
Group Assignment #1📋

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#27] [Group Assignment]
## Due 7/3 at 3:15pm PST

Post the link to your group repository alongside the group members here to complete [group assignment #1](https://albertkun.github.io/24SU-ASIAAM-191A/assignments/week1/group_assignment/)

### Example submission

Albert, Yoh, Nina: https://github.com/albertkun/my-example-project/

**Comment by sjeyakum** [comment]
Michelle, Sadhana: https://github.com/zhamic7/aa191a-su24

**Comment by pwinnie2026** [comment]
Kiana, Nellie, Paige: https://github.com/pwinnie2026/group-ate

**Reply by albertkun** [reply]
> Kiana, Nellie, Paige: [https://github.com/pwinnie2026/group-ate](url)

Nice name!! I am hungry to learn more! 🥘

**Comment by chrissanrow** [comment]
Chris, Kienna: https://github.com/chrissanrow/191A_LGBTQintl

**Comment by giaari15** [comment]
Gianna, Justine: https://github.com/stinest/24su-191a-group4

**Comment by Keiya-Wada** [comment]
Keiya, Kaila: https://github.com/Keiya-Wada/asiaam-japanese

**Comment by nesslee123** [comment]
Ness & Sachi: https://github.com/nesslee123/disability-project.git

**Comment by caitlynn-hauw** [comment]
Caitlynn and Jasmine: https://github.com/jasminereyz/Post-Incarceration-Health.git
